packages/console/src/types/applications.ts

```diff
@@ -18,9 +22,32 @@ export enum SupportedSdk {
   Go = 'Go',
 }
 
+/** @deprecated */
 export const applicationTypeAndSdkTypeMappings = Object.freeze({
   [ApplicationType.Native]: [SupportedSdk.iOS, SupportedSdk.Android],
   [ApplicationType.SPA]: [SupportedSdk.React, SupportedSdk.Vue, SupportedSdk.Vanilla],
   [ApplicationType.Traditional]: [SupportedSdk.Next, SupportedSdk.Express, SupportedSdk.Go],
   [ApplicationType.MachineToMachine]: [],
 } as const);
+
+/**
+ * All application guide categories, including all 4 existing application types,
+ * plus the "featured" category.
+ */
+/* eslint-disable import/no-unused-modules */
+export const allAppGuideCategories = Object.freeze([
+  'featured',
```

Comment:
should we capitalize this category name? others are all capitalized.

Reply:
I think we only capitalize types, and this is not a type but a static constant.